lib/exec.js

```diff
@@ -10,7 +10,7 @@ module.exports = function exec(script) {
 
 	script.env = script.env || {};
 
-	var env = objectAssign(process.env, script.env);
+	var env = objectAssign({}, script.env, process.env);
```

Comment:
You are right, the change was breaking. So this is a good question, at what version number this update should be released.

But I was following the logic "not tests – behavior is not defined", so I considered this as change for good.